shorts_pipeline/modules/uploader.py

```
@@ -0,0 +1,72 @@
+
+
+class Uploader:
+    def __init__(self, config: dict):
+        self.cfg = config
+        self.youtube = self._build_client()
+
+    def _build_client(self):
+        token_file = self.cfg["upload"]["youtube_token_file"]
+        try:
+            creds = Credentials.from_authorized_user_file(token_file, SCOPES)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)